connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1739,18 +1845,60 @@ ble_error_t Gap::setAdvertisingParameters(
     const AdvertisingParameters &params
 )
 {
+    tr_info("Set advertising parameters for set %d: "
+            "_advType=%s "
+            "_minInterval=%lums "
```

Comment:
Can you use the macro constant (https://en.cppreference.com/w/c/types/integer) . `%lu` may not work well on all toulchain.

Reply:
Do you mean simply `%u` ?